web/src/components/storage/ProposalFailedInfo.tsx

```diff
@@ -42,59 +57,43 @@ function Description({ partitions, booting }) {
     );
   }
 
-  const mountPaths = newPartitions.map((p) => partitionUtils.pathWithSize(p));
-  const msg1 = booting
-    ? sprintf(
-        // TRANSLATORS: %s is a list of formatted mount points with a partition size like
-        // '"/" (at least 10 GiB), "/var" (20 GiB) and "swap" (2 GiB)'
-        // (or a single mount point in the singular case).
-        n_(
-          "It is not possible to allocate the requested partitions for booting and for %s.",
-          "It is not possible to allocate the requested partitions for booting, %s.",
-          mountPaths.length,
-        ),
-        formatList(mountPaths),
-      )
-    : sprintf(
-        // TRANSLATORS: %s is a list of formatted mount points with a partition size like
-        // '"/" (at least 10 GiB), "/var" (20 GiB) and "swap" (2 GiB)'
-        // (or a single mount point in the singular case).
-        n_(
-          "It is not possible to allocate the requested partition for %s.",
-          "It is not possible to allocate the requested partitions for %s.",
-          mountPaths.length,
-        ),
-        formatList(mountPaths),
-      );
-
   return (
     <>
-      <Content component="p">{msg1}</Content>
+      <Content component="p">
+        {sprintf(
+          // TRANSLATORS: %s is a list that includes "boot partition" and one or
+          // more formatted mount points with size like: '"/" (at least 10 GiB),
+          // "/var" (20 GiB), and "swap" (2 GiB)'.
+          _("It is not possible to allocate space for %s."),
```

Comment:
I'm not against this, but I just wanted to mention that putting something like "boot partition" in the same list that the mountpoints can be a bit challenging for translations (even for the translators to understand how this and the separate string "boot partition" work together).

I'm not sure if something like this would be better for i18n:

"It is not possible to allocate space for the boot partition and for %s."
and
"It is not possible to allocate space for %s."

Reply:
Indeed, your suggestion is better and has a lot of sense. Thanks.